packages/sdk/src/utils.ts

```diff
@@ -0,0 +1,4 @@
+
+export function standardizeString(name: string): string {
+  return name.replace(/[^a-z0-9]+/gim, '').toLowerCase();
```

Comment:
Is this tested? This should just remove every char that is not alphanumeric, right?

Reply:
this code is from @SurgeCode , I didnt test it